index.js

```diff
@@ -3,17 +3,18 @@
 import _get from 'lodash/get'
 
 const SLACK_TOKEN = process.env.SLACK_TOKEN
-const BOT_NAME = 'GoodDolar Support'
-const AMPLITUDE_KEY = process.env.AMPLITUDE_KEY
+const BOT_NAME = 'GoodDollar Support'
 const AMPLITUDE_SECRET = process.env.AMPLITUDE_SECRET
 const SENTRY_PROJECT_ID = process.env.SENTRY_PROJECT
 const SENTRY_KEY = process.env.SENTRY_KEY
 const REAMAZE_USER = process.env.REAMAZE_USER
 const REAMAZE_TOKEN = process.env.REAMAZE_TOKEN
-const TRAVIS_TOKEN = process.env.TRAVIS_TOKEN
 const SLACK_MONITORING_WEBHOOK = process.env.SLACK_MONITORING_WEBHOOK
+const GITHUB_DAPP_WORKFLOW_ID = process.env.GITHUB_DAPP_WORKFLOW_ID
```

Comment:
I see we have now a lot of the env vars. Maybe it's time to create config.js file using convict to parse them from dotenv.

Reply:
I try to keep the scope of this task as minimal as possible
this service is small isolated and does not change very often